### PR TITLE
Issue/5 update screenshots

### PR DESCRIPTION
**Pull request by nathaned**
* Screenshots updated
    * now higher resolution full screenshots
    * all screenshots same aspect ratio
* fixed typo `Addd` -> `Add`
* fixed issue in creator where clicking anywhere would hide the initial dialog
* also removes `print.js` from `player.html` since it's now bundled into `player.js`

### CODE REVIEW

File: src/creator.coffee [hunk 1]
```diff
@@ -17,8 +17,6 @@ CrosswordCreator.directive 'selectMe', ['$timeout', '$parse', ($timeout, $parse)
 			if value is true
 				$timeout ->
 					$(element[0]).focus().select()
-		element.bind 'blur', ->
```
Comment (iturgeon) [inline src/creator.coffee]: is this a problem?

File: src/player.html [hunk 2]
```diff
@@ -11,9 +11,6 @@
 		<!-- REQUIRED MATERIA JAVASCRIPT -->
 		<script src="materia.enginecore.js"></script>
 
-		<!-- OPTIONAL MATERIA JAVASCRIPT -->
-		<script src="print.js" data-embed></script>
```
Comment (iturgeon) [inline src/player.html]: confirmed, still prints and this file is gone now. I assume it was bundled into the player.

File: src/creator.coffee (hunk 1 above)
Comment (nathaned) [reply src/creator.coffee]: This modal automatically pops up when you create a new widget:
Screen Shot 2019-04-09 at 4 02 55 PM
Before:
if you click *anything* other than the input field, it'll close the modal. That includes clicking the modal, but not on the input field, or even selecting a different tab/window.

After:
Modal only closes when:
   * gray background clicked 
   * enter pressed on input box 
   * "get started" clicked

Comment (iturgeon) [reply src/creator.coffee]: I see. I agree, sounds better this way.

**Pull request merged**